ml_service/pipelines/build-model-container-image.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+    agent { label 'master' }
+    environment {
+        ML_IMAGE_FOLDER         = 'imagefiles'
+        IMAGE_NAME              = 'mlmodelimage'
+        MODEL_NAME = 'diabetes_regression_model.pkl'
+        MODEL_VERSION = '1'
+        SCORE_SCRIPT = 'scoring/score.py'
+        RESOURCE_GROUP = "${RESOURCE_GROUP}"
+        WORKSPACE_NAME = "${WORKSPACE_NAME}"
+        ML_CONTAINER_REGISTRY   = "${ML_CONTAINER_REGISTRY}"
+    }
+    stages {
+        stage('initialize') {
+            steps {
+                echo 'Remove the previous one!'
+            }
+            post {
+                always {
+                    deleteDir() /* clean up our workspace */
+                }
+            }
+        }
+        stage('generate_dockerfile') {
+            steps {
+                echo "Hello build ${env.BUILD_ID}"
+                /*checkout scm*/
+                checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc76']],
+                    userRemoteConfigs: [[url: 'https://github.com/Merlion-Crew/MLOpsPython.git/']]])
+
+                azureCLI commands: [[exportVariablesString: '/id|SUBSCRIPTION_ID', script: "az account show"]], principalCredentialId: "${AZURE_SP}"
+                
+                sh '''#!/bin/bash -ex
+                    source /home/azureuser/anaconda3/bin/activate mlopspython_ci
```

Comment:
Seems like we need to be very explicit about having this dependency (`anaconda`) installed in the jenkins server to be able to execute this. cc/ @sundar-swetha in case it needs to be approved.